### DISCUSSION TITLE
Model prediction

### DISCUSSION THREAD

**Discussion by rondon-d** [inlabru-org/inlabru#215] [General]
Hello team 

Going back to the Gorilla nest example that is presented on the website.
Where the output is the nest locations and it is of interest to study the effect of different covariates

Is it possible to build a lgcp in inlabru that allow me to find plausible locations of non-observed nest, given a known set of covariates across the area of interest?

My thought is that when I build the model
I am informing it that I know where the nests are and are not
so if I plan to build a model to do prediction (input: coordinates -> output: prob of a nest), It will be difficult to estimate unseen data. 

I hope that I am clear, sorry if not. 
I am still processing the idea

best wishes :)

**Comment by finnlindgren** [comment]
You'd need to formulate a model for the conditional probability of observing a nest given that it is there, similarly to the detection probability models in the distance sampling cases.  That model can then depend on covariates.
You would need to carefully consider identifiability issues; in the distance sampling case, that's done by assuming that the detectability at ditance zero is 1. You would need some kind of constraint like that in your case as well.

But maybe I'm misunderstanding, and you're just talking about the _prediction_ step for a _new_ region?

**Comment by rondon-d** [comment]
Hello, 
Thank you for the replay, I understand your comment. 
My plan is to make predictions over the same study region (same mesh). 
to put it in the perspective of the gorilla nest example: I am looking for more nests and I want to identify probable locations based on the covariates. 

Thank you for your time :)

**Reply by finnlindgren** [reply]
Do you mean something like this example? https://inlabru-org.github.io/inlabru/articles/2d_lgcp_plotsampling.html
There, we assume to have perfect point detection within each plot, and no observations outside the plots, so that the results there are predictions of probable nest locations, based on the model estimated on the information within the plots.  FOr simplicity of presentation, the example doesn't use any covariates, but incorporating that would work exactly the same as in https://inlabru-org.github.io/inlabru/articles/2d_lgcp_covars.html
